lab-lacy/src/main.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const reader = require('./lib/reader');
+
+const mobyDickPath = `${__dirname}/data/moby-dick.txt`;
+const sherlockPath = `${__dirname}/data/sherlock.txt`;
+const ulyssesPath = `${__dirname}/data/ulysses.txt`;
+
+const files = [mobyDickPath, sherlockPath, ulyssesPath];
+
+const printCharacters = () => {
```

Comment:
You left printCharacters blank so when I run main.js nothing is actually printed beside your console.log. Not going to take off points, but you can pass in file and log that so we can see the contents of your files, i.e.

const printCharacters = (file) => {
  console.log(file);
};